README.md

```diff
@@ -9,12 +9,22 @@
 [![docs](https://img.shields.io/badge/docs-pkgdown-blue.svg)](https://icbi-lab.github.io/immunedeconv)
 
 ## Basic usage
+
+Deconvolution of human data:
+
 ```R
 immunedeconv::deconvolute(gene_expression_matrix, "quantiseq")
 ```
 
+Deconvolution of mouse data:
+
+```R
+immunedeconv::deconvolute_mouse(gene_expression_matrix, "quantiseq")
```

Comment:
is quantiseq a good example here?

Reply:
No it's a mistake
I got confused with the section where I make the conversion of gene names